component/Badge/badge.css

```diff
@@ -0,0 +1,97 @@
+.badge_container {
+    position: relative;
+}
+
+.badge {
+    position: absolute;
+    background-color: var(--primary-color);
+    left: 9.5rem;
+    bottom: 1.1rem;
+    width: 2rem;
+    height: 2rem;
+    border: 3px solid var(--white-color);
+    border-radius: 50%;
+}
+
+.badge_md_size {
+    width: 1.5rem;
+    height: 1.5rem;
+    background-color: var(--danger-alert-color);
+}
+
+.badge_sm_size {
+    height: 1rem;
+    width: 1rem;
+}
+
+.badge_med {
+    position: absolute;
+    left: 7.5rem;
+}
+
+.badge_small {
+    position: absolute;
+    left: 5rem;
+    background-color: var(--success-alert-color);
+}
+
+.badge_xs {
+    position: absolute;
+    left: 2.5rem;
+    bottom: 0.5rem;
+    background-color: var(--warning-alert-color);
+}
+
+.icons_border {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    gap: 5rem;
+}
```

Comment:
It does not make sense to name property as icomn_border, when you are not using border property at all in that class. This could have been just `icon`.

Reply:
In addition to what Tahir said, you can just use _flex-flow: row wrap;_ instead of _flex-wrap: wrap;_ and
    _flex-direction: row;_